### PR TITLE
Update summary

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

**Comment by nanonyme** [comment]
Where's this string from?

File: com.valvesoftware.Steam.appdata.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <id type="desktop">com.valvesoftware.Steam.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
   <name>Steam</name>
-  <summary>Manage and play games distributed by Steam</summary>
+  <summary>Play popular games and the latest releases</summary>
```
Comment (sat3ll) [inline com.valvesoftware.Steam.appdata.xml]: The previous summary seems a better fit although I understand your motivation to change it to this. The "pt_BR" and "es" languages are literal translations of your new proposal for the "english" summary. In my view it is not necessary to "harmonise"/make them all the same literal translation. Literal translations often make the phrase sound "strange". As long the intended meaning/message is conveyed, there's no problem for the translations to "diverge" literally.

**Review by sat3ll — COMMENTED** [review]
Literal translation of "es" and "pt_BR"

File: com.valvesoftware.Steam.appdata.xml (hunk 1 above)
Comment (nanonyme) [reply com.valvesoftware.Steam.appdata.xml]: Possibly "Play latest releases of popular games" if you want the change. But again, where's these strings from? Are they from Valve?

Comment (sat3ll) [reply com.valvesoftware.Steam.appdata.xml]: I can't find them anywhere on steam site (set to pt_BR). Git blame points the "es" and "pt_BR" translations to @LeandroStanger so I assume he translated them himself.

**Pull request closed without merging**